compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -353,10 +356,6 @@ void addCPUBufferOpsTileAndVectorizePipeline(OpPassManager &passManager,
     nestedModulePM.addNestedPass<func::FuncOp>(createCSEPass());
   }
 
-  // Run IREE specific passes before vector lowering expert.
-  nestedModulePM.addNestedPass<func::FuncOp>(
```

Comment:
For all the pipelines from where we are removing this pass, we should set the tile and distribute to not generate the loops to begin with?

Reply:
I thought that was the case, did I miss something?

Reply:
`linalg::DistributionMethod::CyclicNumProcsEqNumIters` is now the default for CPU and only VMVX should have the loops